wsgi/odyssy/odyssy/settings.py

```diff
@@ -36,6 +36,7 @@
 DEBUG = not ON_OPENSHIFT
 
 ALLOWED_HOSTS = [
+    '127.0.0.1',
```

Comment:
Yeah, I agree with you. But it would be better is we do minimal changes while deploying the website on Apache. Moreover, keeping it generic doesn't have practical use. Most of us will be testing on localhost.